tests/package.json

```diff
@@ -4,6 +4,6 @@
     "chai": "^4.3.4",

```

Comment:
Please kindly remove your changes. It has already been updated on the master branch.

Reply:
Also, delete the `chromedriver.exe` file you added below, and please refrain from **modifying** any code in the `tests` folder. Thank you.